proto/dwtools/abase/l0.test/Long.test.s

```diff
@@ -11778,56 +11778,461 @@ function longAreRepeatedProbe( test )
 function longAllAreRepeated( test )
 {
 
+  /* Without evaluator */
+
+  test.case = 'empty array';
   var got = _.longAllAreRepeated( [] );
-  test.identical( got, true );
+  var expected = true;
+  test.identical( got, expected );
 
+  test.case = 'identical numbers of array';
   var got = _.longAllAreRepeated( [ 1, 1 ] );
-  test.identical( got, true );
+  var expected = true;
+  test.identical( got, expected );
 
+  test.case = 'single number of array';
   var got = _.longAllAreRepeated( [ 1 ] );
-  test.identical( got, false );
+  var expected = false;
+  test.identical( got, expected );
 
+  test.case = 'some numbers of array are identical';
   var got = _.longAllAreRepeated( [ 1, 2, 2 ] );
-  test.identical( got, false );
+  var expected = false;
+  test.identical( got, expected );
+
+  /**/
+
+  test.case = 'identical strings of array';
```

Comment:
Please, insert this test case `after `identical numbers of array`.`Number` and `String` are primitive types.